Pilot/pilotTools.py

```diff
@@ -311,6 +301,11 @@ def getSubmitterInfo(ceName):
 
     # # CEs
 
+    # HTCondor
+    if "HTCONDOR_JOBID" in os.environ:
+        flavour = "HTCondorCE"
+        pilotReference = "htcondorce://" + ceName + "/" + batchSystemJobID
```

Comment:
Is `batchSystemJobID` going to be always `None` in this case? If that's the case, as it seems to be, apart that this as it is would raise a `TypeError`, we would end up anyway having a number of `pilotReference`s that are all the same.  If there's no `batchSystemID` identifiable, we better report `None` as `pilotReference`: in this case the `pilotUUID` would be used, which is guaranteed to be there and be unique.